src/Examine/Logging/LogEntry.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+namespace Examine.Logging
+{
+    public class LogEntry
```

Comment:
Last thing - but I'll fix up when I merge - this should be a struct else we are allocating to the heap everytime we attempt to log.